docs/connect.md

```diff
@@ -50,8 +50,8 @@ Finally, connect to the party in your client code:
 import { useFireproof } from 'use-fireproof'
 import { connect } from '@fireproof/partykit'
 
-const { database } = useFireproof('my-app-database-name')
-const connection = connect.partykit(database, process.env.PUBLIC_PARTYKIT_HOST!)
+const { ledger } = useFireproof('my-app-ledger-name')
```

Comment:
gotta check the react api here

Reply:
marking this resolved, we have a cleanup of this file planed